reslib/pom.xml

```diff
@@ -47,10 +47,10 @@
 									<goal>wget</goal>
 								</goals>
 								<configuration>
-									<url>https://github.com/liferay/liferay-portal/raw/0f75db5b5c6a8a7c6b07295cc12acdf362bfb1b4/portal-web/third-party/alloy-${alloyui.zip.version}.zip</url>
+									<url>https://github.com/liferay/liferay-portal/raw/6.2.x/portal-web/third-party/alloy-${alloyui.zip.version}.zip</url>
```

Comment:
Right, things will work fine now, but in the future, if the zip is upgraded in the `6.2.x` branch, then our build will fail because the md5sum will no longer match. This has happened before ([FACES-2476](https://issues.liferay.com/browse/FACES-2476)). Please ping me on Skype if I'm not being clear enough.